### PR TITLE
Added impinj fixedreader + R700 model files

### PR DESCRIPTION
**Pull request by kwennerImpinj**
### Thank you for contributing to the Azure IoT Plug and Play Models repository

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

The repository pipeline will ensure minimum validation of incoming DTDL models.  

- PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).
- The same validation can be run locally via the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet#device-model-repository-client) CLI.

## Requested info template for model(s) submission

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve Plug and Play.

:star2: Please replace the markdown comment examples with your own values.

### CODE REVIEW

File: dtmi/impinj/fixedreader/fixedreader-1.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "@id": "dtmi:impinj:FixedReader;1",
+  "@type": "Interface",
+  "displayName": "Pnp Bridge Root Interface Model for Impinj R700",
+  "contents": [
+    {
+      "@type": "Component",
+      "name": "R700",
+      "schema": "dtmi:impinj:R700;13"
```
Comment (rido-min) [inline dtmi/impinj/fixedreader/fixedreader-1.json:9]: update to use the full id

File: dtmi/impinj/fixedreader/r700-13.json [hunk 2]
```diff
@@ -0,0 +1,3882 @@
+{
+  "@id": "dtmi:impinj:R700;13",
```
Comment (rido-min) [inline dtmi/impinj/fixedreader/r700-13.json:2]: based on the file path, seems this id is missing the `fixedreader` segment in the dtmi.

**Review by rido-min — CHANGES_REQUESTED** [review]
can you review the IDs/folder path for the component?

File: dtmi/impinj/fixedreader/fixedreader-1.json (hunk 1 above)
Comment (kwennerImpinj) [reply dtmi/impinj/fixedreader/fixedreader-1.json]: updated!  (moved file to same folder)

File: dtmi/impinj/fixedreader/r700-13.json (hunk 2 above)
Comment (kwennerImpinj) [reply dtmi/impinj/fixedreader/r700-13.json]: updated!  moved r700-13.json to same folder /dtmi/impinj

**Review by rido-min — APPROVED** [review]

**Review by daisukeiot — APPROVED** [review]
Worked with Kevin to refine some details, and tested/validated with IoT Explorer, DTDL Validator, etc.

**Review by daisukeiot — APPROVED** [review]

**Review by rido-min — APPROVED** [review]

**Pull request merged**